SU2_CFD/include/CMarkerProfileReaderFVM.hpp

```diff
@@ -111,6 +114,7 @@ class CMarkerProfileReaderFVM {
    */
   CMarkerProfileReaderFVM(CGeometry      *val_geometry,
                           CConfig        *val_config,
+                          CSolver        **val_solver,
                           string         val_filename,
```

Comment:
I'm very sorry but this is not the way. Having generic code that is orthogonal to each other is a good thing, and we need to move in that direction, always.

Prepare the column names and default values outside of this class, in the method that uses it for example.
Then give this class a method to pass in that information, when it is not passed it can have default values like "Var1" "Var2", etc.

Reply:
I would have preferred that we have the number of solvers available somewhere, and we can then take the numvars of each solver and do a summation. I now need numvar_turb, later also numvar_scalar, probably others later as well (radiation?). 
Do we have the number of solvers available?

Reply:
I removed val_solver

Reply:
The solver container has a maximum number of solvers and there are fixed positions for different types of solver.
You could, for example, add a method to solvers to be able to ask the solver "do you like inlet profiles? what are the name of the variables you want from it? what default values should they have?"
Then you can loop over all possible solvers and ask them that (if they exist i.e. not nullptr), that gets passed down to the profile reader (as lists of strings, whose content is irrelevant for a class that just reads a table).